Helpers/CollectCmsConfigHelper.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Alma\MonthlyPayments\Helpers;
+
+use Alma\API\Exceptions\AlmaException;
+use Alma\MonthlyPayments\Helpers\Exceptions\AlmaClientException;
+use Magento\Framework\App\Cache\TypeListInterface;
+use Magento\Framework\App\Config\Storage\WriterInterface;
+use Magento\Framework\App\Helper\Context;
+use Magento\Framework\Serialize\SerializerInterface;
+
+class CollectCmsConfigHelper extends ConfigHelper
+{
+    // Path to the configuration that stores the last time we sent the collect URL to Alma in system.xml
+    const SEND_COLLECT_URL_STATUS_PATH = 'send_collect_url_status';
+
+    const COLLECT_URL = '/V1/alma/config/collect';
+
+    private $almaClient;
+    private $logger;
+
+    /**
+     * @param Context $context
+     * @param StoreHelper $storeHelper
+     * @param WriterInterface $writerInterface
+     * @param SerializerInterface $serializer
+     * @param TypeListInterface $typeList
+     * @param AlmaClient $almaClient
+     * @param Logger $logger
+     */
+    public function __construct(
+        Context             $context,
+        StoreHelper         $storeHelper,
+        WriterInterface     $writerInterface,
+        SerializerInterface $serializer,
+        TypeListInterface   $typeList,
+        AlmaClient          $almaClient,
+        Logger              $logger
+    )
+    {
+        parent::__construct($context, $storeHelper, $writerInterface, $serializer, $typeList);
+        $this->almaClient = $almaClient;
+        $this->logger = $logger;
+    }
+
+    /**
+     * Send_collect_url_status getter
+     *
+     * @return string|null
+     */
+    public function getSendCollectUrlStatus(): ?string
+    {
+        return $this->getConfigByCode(self::SEND_COLLECT_URL_STATUS_PATH);
+    }
+
+    /**
+     * Send url to Alma
+     *
+     * @return void
+     */
+    public function sendIntegrationsConfigurationsUrl(): void
+    {
+        try {
+            $this->almaClient->getDefaultClient()->merchants->sendIntegrationsConfigurationsUrl(self::COLLECT_URL);
```

Comment:
In my integration, I sent the absolut url. It isn't better to send all the url to Alma to don't add intelligence for Alma ?